client/my-sites/plugins/plugin-details-sidebar/style.scss

```diff
@@ -36,6 +36,11 @@
 		background: transparent;
 	}
 
+	.plugin-details-sidebar__deprecated-notice {
+		font-size: $font-body-extra-small;
+		color: var(--color-warning);
+	}
+
```

Comment:
This seems unused.

Reply:
Yes, that is from a previous commit. Well spotted! I am going to remove that